sleap/gui/commands.py

```diff
@@ -2029,6 +2033,34 @@ def ask(context: CommandContext, params: dict) -> bool:
 
         return len(filename) > 0
 
+class LinkVideoToSession(EditCommand):
+    topics = [UpdateTopic.video]
```

Comment:
A bit confusing, it seems like the update topic would be the video, but actually, we are making changes to the `RecordingSession`, not to the `Video`.
```suggestion
    topics = [UpdateTopic.sessions]
```